.github/workflows/test.yml

```diff
@@ -34,6 +39,17 @@ jobs:
       - name: cargo test --doc
         run: cargo test --locked --all-features --doc
   minimal:
+    # This action chooses the oldest version of the dependencies to ensure that this crate is
+    # compatible with the minimal version that this crate and its dependencies require.
+    # This will pickup issues where this create relies on functionality that was introduced
+    # later than the actual version specified (e.g. when we choose just a major version, but a
```

Comment:
nit: "e.g." and "i.e." should basically always be followed by a comma (according to _most_ style guides).